test/logflare/sources_test.exs

```diff
@@ -8,22 +8,21 @@ defmodule Logflare.SourcesTest do
   alias Logflare.Google.BigQuery.GenUtils
   alias Logflare.Source.RecentLogsServer, as: RLS
   alias Logflare.Source
-  @moduletag :this
 
-  setup do
-    u = Users.get_by(email: System.get_env("LOGFLARE_TEST_USER_WITH_SET_IAM"))
-    s = insert(:source, token: Faker.UUID.v4(), rules: [], user_id: u.id)
+  @tag :failing
+  describe "Sources" do
+    setup do
+      u = Users.get_by(email: System.get_env("LOGFLARE_TEST_USER_WITH_SET_IAM"))
+      s = insert(:source, token: Faker.UUID.v4(), rules: [], user_id: u.id)
```

Comment:
can simplify this to:

```
user = insert(:user)
source = insert(:source, user: user)
```

also, would defer towards using fully qualified variable  names where possible.

Reply:
can start to tackle it in a cleanup PR 👍